Cargo.toml

```diff
@@ -37,13 +36,13 @@ members = [
 ]
 
 [dev-dependencies]
-actix = "0.8.2"
+actix = "0.9.0-alpha.2"
```

Comment:
Should we use a pre-release version as dependency? I think 0.9 is going to be released soon https://github.com/actix/actix/commit/8ad0a991b99b1a03f581c1ce33a2c86c5e2c9dac

Reply:
just got released